### ISSUE TITLE
uuv_description does not build in debs on Kinetic

### ISSUE THREAD

**Issue by tfoote** [uuvsimulator/uuv_simulator#342]
It looks like a missing dependency on gazebo.

http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__uuv_descriptions__ubuntu_xenial_amd64__binary/15/console
```
CMake Error at CMakeLists.txt:9 (find_package):
  By not providing "Findgazebo.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "gazebo", but
  CMake did not find one.

  Could not find a package configuration file provided by "gazebo" with any
  of the following names:

    gazeboConfig.cmake
    gazebo-config.cmake

  Add the installation prefix of "gazebo" to CMAKE_PREFIX_PATH or set
  "gazebo_DIR" to a directory containing one of the above files.  If "gazebo"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Comment by musamarcusso** [comment]
I am making alteration to solve this at #345. I used an old example to use `find_package(gazebo REQUIRED)` instead of the package `gazebo_dev` for a long time. Since the builds were still passing, I couldn't catch this before. I will merge it as soon all tests pass and make a new release.

**Comment by musamarcusso** [comment]
Issue fixed on release `0.6.10`
